src/adapters/interfaces/IEOFeedRegistryAdapter.sol

```diff
@@ -10,6 +10,7 @@ import { IEOFeedAdapter } from "./IEOFeedAdapter.sol";
  */
 interface IEOFeedRegistryAdapter {
     // V3 AggregatorV3Interface
+    function removeFeedAdapter(address base, address quote) external;
```

Comment:
move it one line above pls if it is needed in interface
(before // V3 AggregatorV3Interface)